bobtask/task.go

```diff
@@ -34,6 +34,9 @@ type Task struct {
 
 	// InputDirty is the representation read from a bobfile.
 	InputDirty string `yaml:"input"`
+	// InputAdditionalIgnores is a list of ignores
```

Comment:
This comment is part is not so clear to me: "usually the child targets" If it's usually, then what else when is not usual?